Formula/m/mysql-client.rb

```diff
@@ -35,7 +35,26 @@ class MysqlClient < Formula
 
   uses_from_macos "libedit"
 
+  # std::string_view is not fully compatible with the libc++ shipped
+  # with ventura, so we need to use the LLVM libc++ instead.
+  on_ventura :or_older do
+    depends_on "llvm@18"
+    fails_with :clang
+  end
```

Comment:
Does this not work with LLVM 19?

Reply:
I didn't have any success when I tried on version 19 earlier. But I can keeping poking at it.

Reply:
Latest `llvm` fails on Ventura with

```
/tmp/mysql-client-20250102-8365-ekydyl/mysql-9.1.0/libs/mysql/serialization/archive.h:84:43: error: a template argument list is expected after a name prefixed by the template keyword [-Wmissing-template-arg-list-after-template-kw]
     84 |     return Archive_derived_type::template get_size(std::forward<Type>(arg));
        |                                           ^
```

I think there was a comment specifically about this at one point, but there's been lots of iterations in the mean time.

Reply:
Weird. Seems fine then, thanks!